### PR TITLE
fix a couple of issues for useful terms section

### PR DESCRIPTION
**Pull request by greenhsu123**
fix a couple of issues for useful terms section

### CODE REVIEW

File: docs/practitioner/glossery.md [hunk 1]
```diff
@@ -221,11 +221,9 @@ sidebar_position: 7
...
-* **Embodied carbon** (also known as "embedded carbon") - The amount of carbon pollution emitted during the creation and disposal of a device.
+* [Value chain emissions](https://www.cisl.cam.ac.uk/education/graduate-study/pgcerts/value-chain-defs) - These are scope 3 emissions according to the GHG protocol, and the most significant source of emissions. They encompass the full range of activities needed to create a product or service, from conception to distribution.
+* [Energy proportionality](https://learn.greensoftware.foundation/practitioner/energy-efficiency#energy-proportionality) - Measures the relationship between power consumed by a computer and the rate at which useful work is done (its utilization).
+* [Static power draw](https://learn.greensoftware.foundation/practitioner/energy-efficiency#static-power-draw) - This refers to how much electricity is drawn when a device is in an idle state. 
+* [Embodied Carbon](https://learn.greensoftware.foundation/practitioner/hardware-efficiency#embodied-carbon) (also known as "embedded carbon") - The amount of carbon pollution emitted during the creation and disposal of a device.
```
Comment (dubrie) [inline docs/practitioner/glossery.md:229]: Minor nit, and it should still resolve in most browsers, but the anchor links all redirect to "/#ANCHOR_TAG" when clicked. In the name of reducing overall calls and work for the server, we should be explicitly link to the directory in the following:

- https://learn.greensoftware.foundation/practitioner/energy-efficiency/#energy-proportionality
- https://learn.greensoftware.foundation/practitioner/energy-efficiency/#static-power-draw
- https://learn.greensoftware.foundation/practitioner/hardware-efficiency/#embodied-carbon

**Review by dubrie — APPROVED** [review]
Approved, with a minor change suggestion

File: docs/practitioner/glossery.md (hunk 1 above)
Comment (holanita) [reply docs/practitioner/glossery.md:229]: I have updated these urls in the content that I am now submitting to the LF. pls handle the PR.

Comment (greenhsu123) [reply docs/practitioner/glossery.md:229]: I think we already are except value chain emissions (which we don't have a direct link to gsf website)? apologies, if i have missed sth completely...

Comment (dubrie) [reply docs/practitioner/glossery.md:229]: My comment here was calling out the missing "/" in the URL string. For example, the carbon-awareness link currently links to:

https://learn.greensoftware.foundation/practitioner/carbon-awareness#carbon-intensity

Note there is no "/" between the path and the anchor link: `carbon-awareness#carbon-intensity`. When you click a link like that, our site's web server redirects it to the following URL:

https://learn.greensoftware.foundation/practitioner/carbon-awareness/#carbon-intensity

Note the "/" is present at the end of the path and before the anchor link: `carbon-awareness/#carbon-intensity`. 

This is quite minor and doesn't block any functionality or links.

Comment (holanita) [reply docs/practitioner/glossery.md:229]: guys it's all sorted in the google docs. can you update the website as well so we can close this and announce 'code freeze' :)

**Review by jawache — APPROVED** [review]

**Comment by jawache** [comment]
All approved. @greenhsu123 can merge once changes been made.

**Pull request merged**